specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/machineLearningServices.json

```diff
@@ -0,0 +1,5550 @@
+          "$ref": "#/definitions/KeyVaultProperties",
+          "description": "Customer Key vault properties."
+        }
+      },
+      "required": [
+        "status",
+        "keyVaultProperties"
+      ],
+      "type": "object"
+    },
+    "KeyVaultProperties": {
+      "properties": {
+        "keyVaultArmId": {
+          "description": "The ArmId of the keyVault where the customer owned encryption key is present.",
+          "type": "string"
+        },
+        "keyIdentifier": {
+          "description": "Key vault uri to access the encryption key.",
+          "type": "string"
+        },
+        "identityClientId": {
+          "description": "For future use - The client id of the identity which will be used to access key vault.",
+          "type": "string"
+        }
+      },
+      "required": [
+        "keyIdentifier",
+        "keyVaultArmId"
+      ],
+      "type": "object"
+    },
+    "LinkedServiceResponse": {
+      "description": "Linked service.",
+      "type": "object",
+      "x-ms-azure-resource": true,
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "ResourceId of the link of the linked service.",
+          "readOnly": true
+        },
+        "name": {
+          "type": "string",
+          "description": "Friendly name of the linked service.",
+          "readOnly": true
+        },
+        "type": {
+          "type": "string",
+          "description": "Resource type of linked service.",
+          "readOnly": true
+        },
+        "location": {
+          "description": "location of the linked service.",
+          "type": "string"
+        },
+        "identity": {
+          "$ref": "#/definitions/Identity"
+        },
+        "properties": {
+          "$ref": "#/definitions/LinkedServiceProps"
+        }
+      }
+    },
+    "LinkedServiceRequest": {
+      "type": "object",
+      "description": "object used for creating linked service.",
+      "properties": {
+        "name": {
+          "description": "Friendly name of the linked service",
+          "type": "string"
+        },
+        "location": {
+          "description": "location of the linked service.",
+          "type": "string"
+        },
+        "identity": {
+          "$ref": "#/definitions/Identity"
+        },
+        "properties": {
+          "$ref": "#/definitions/LinkedServiceProps"
+        }
+      }
+    },
+    "LinkedServiceProps": {
+      "type": "object",
+      "description": "LinkedService specific properties.",
+      "properties": {
+        "linkedServiceResourceId": {
+          "description": "ResourceId of the link target of the linked service.",
+          "type": "string"
+        }
+      }
+    },
+    "LinkedServiceList": {
+      "readOnly": true,
+      "type": "array",
+      "description": "An array of linked services",
+      "items": {
+        "$ref": "#/definitions/LinkedServiceResponse"
+      }
+    },
```

Comment:
This response should be wrapped in an object like this:
{
"value": [] // <- the array goes here
}

Reply:
Like this https://github.com/Azure/azure-rest-api-specs/blob/13310949816c62c0f3f4dcea23ab2479b82686bd/specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/machineLearningServices.json#L1886